### DISCUSSION TITLE
4.38: Release Information

### DISCUSSION THREAD

**Discussion by james-d-elliott** [authelia/authelia#4813] [Announcements]
Released.

**Comment by ServusNL** [comment]
Love the project! But there is quite some time between the pre-release and release? Is this still in development?

**Reply by james-d-elliott** [reply]
Yes, unfortunately my time has been pretty limited lately. In addition the changes in particular to the identity verify process are very delicate. We are getting there though. I've updated the original post with more information.

**Reply by ServusNL** [reply]
Thanks for the update and all the effort put in to the project! Much appreciated! Looking forward to the update.

**Comment by james-d-elliott** [comment]
We will get this released, and we understand this has taken longer than initially anticipated. We cannot generally provide ETA's unfortunately, however as soon as more information is available on a timeline (most likely after everything is merged and ready to test); we will update this discussion.

Unfortunately my time has been rather limited lately due to an increased demand of my time at work (including a lot of overtime) where I'm having to leverage my engineering experience to fix important and time critical issues which have developed due to resignations and a lack of others performing their respective responsibilities. 

I've been working through automating a significant number of these work elements (as well as several of the Authelia chores via generators) via custom developments so my time is freed up to spend my breaks working on things like this rather than just working, making the whole thing a whole lot larger than it needed to be but will in the long term be a benefit to processes and my personal work life balance, and in turn free me up more to work on things like this during breaks. 

These factors and some additional family related stresses has lead to an overall increase in my need to have dedicated relaxation time. These automatons are likely to reduced overall stress and will also free me up at additional times to work on this which would be otherwise spent doing less meaningful activities.

Again this topic, the linked PR's, and the blog post will be updated with any additional information as it becomes available. You can even subscribe to this specific topic if you're interested.

**Reply by speatzle** [reply]
Hi, is it safe to run on master right now? My authelia instance keeps crashing due to #5680 which is fixed but not released. Because the systemd unit from the deb package is messed up and does not automatically restart authelia on failure i have temporarily added a cronjob to start it if it died.

**Reply by james-d-elliott** [reply]
I'd generally recommend against using versions which are not officially published as stable however master should be reasonably safe just ensure you have a backup of your configuration and SQL database configured in the storage area. I would also recommend if you use master that you deliberately update and pin the version you use to a specific commit (to do this with docker you can use the image sha).

As far as the systemd unit I am not sure I'd consider it messed up. We have just not elected to configure a service restart action, leaving it up to the installer. As to if it should be restarted on failure is probably another discussion entirely, but it's relatively easy for end users to configure their services how they want with systemd.

**Comment by james-d-elliott** [comment]
Writing to you all with a fairly large update. A large majority of all of the intended functionality is now implemented, we're almost at the finish line! Also I've hidden all non-specific comments and locked this discussion so you can only react for now so we can keep the information as pointed as possible. 

The following tasks remain outstanding (seems like a lot but most of it effectively sits in the last item):

Task List was moved [here](https://github.com/authelia/authelia/discussions/6206#discussioncomment-7664221).

**Comment by james-d-elliott** [comment]
For all those interested the next beta has been released and the feedback discussions are open. For more information see the relevant pinned topics here: https://github.com/authelia/authelia/discussions/categories/pre-release-feedback
